package/libs/boringssl/patches/010-add-install-target.patch

```diff
@@ -0,0 +1,10 @@
+Index: boringssl-20210608/CMakeLists.txt
+===================================================================
+--- boringssl-20210608.orig/CMakeLists.txt
++++ boringssl-20210608/CMakeLists.txt
```

Comment:
Please add a proper patch header that describes the patch (just as with this commit).

Afterwards, please refresh the patch with `make package/libs/boringssl/refresh V=s` to remove unneeded bloat.